snakemake/deployment/conda.py

```
@@ -831,6 +841,29 @@ def shellcmd_win(self, env_address, cmd):
+            )
+            version_matches = re.findall(r"\d+.\d+.\d+", version)
+            if len(version_matches) != 1:
+                raise WorkflowError(
+                    f"Unable to determine mamba version. 'mamba --version' returned {version}"
+                )
+            else:
+                version = version_matches[0]
+            if Version(version) < Version("2.0.0"):
+                return True
+            else:
+                return False
```

Ruff

Return the condition `Version(version) < Version("2.0.0")` directly

Replace with `return Version(version) < Version("2.0.0")`

(SIM103)